src/issue/screens/new-issue.screen.js

```diff
@@ -90,17 +92,19 @@ class NewIssue extends Component {
       ).then(issue => {
         navigation.navigate('Issue', {
           issue,
+          headerLeft: false,
```

Comment:
Yes I was referring the tab navigator.

Here's a gif of the behavior:
<img width=300 alt="ezgif-4-915da04836" src="https://user-images.githubusercontent.com/9287184/29363767-79c3a5b6-8291-11e7-983d-a51b1d45c851.gif">

When I go back to the "User" tab, I spam the "User" tab item but it doesn't redirect me to my profile screen....